CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ The format of this changelog is based on
 
 ## In progress
 
+  - Changed default value of `config["Solver"]["PartialAssemblyOrder"]` in order to active
+    operator partial assembly by default for all operators in all simulation types.
```

Comment:
activate

Reply:
Fixed in [4ec2ca6](https://github.com/awslabs/palace/pull/168/commits/4ec2ca63727f6797a576803aafa0900c91771eeb).